temporalcli/commandsgen/commands.yml

```diff
@@ -204,10 +204,10 @@ commands:
           The command execution timeout. 0s means no timeout.
 
   - name: temporal activity
-    summary: Complete or fail an Activity
+    summary: Complete, update or fail an Activity
     description: |
-      Update an Activity's state to completed or failed. This marks an Activity
-      as successfully finished or as having encountered an error:
+      Update an Activity's options or update an Activity's state to completed or failed.
+      Updating activity state marks an Activity as successfully finished or as having encountered an error.      
```

Comment:
Please wrap to 80 columns per the "Wrapping" section of the style guide at the top of this file. Same goes for the options and command descriptions.